aten/src/ATen/native/prim_native_functions.cpp

```diff
@@ -37,5 +37,10 @@ Tensor foobar(const Tensor& self, bool arg1, bool arg2, bool arg3) {
   return self;
 }
 
+// Aux function used to test functorch fallback warning
+Tensor _test_functorch_fallback(const Tensor& self, const Tensor& other) {
+  return self;
```

Comment:
nit: we should return self.clone(), otherwise the schema is wrong (the schema promises that the function returns new Tensors.)

Reply:
Thanks, updated.